### ISSUE TITLE
Error on ionic run android

### ISSUE THREAD

**Issue by jQrgen** [tobyn/angular-stripe-checkout#14]
I get this error when i try to build my ionic project for android with the angular-stripe-checkout plugin installed:

```javascript
jqrgen@jQrgenpc:~/stripe-checkout-nodejs-server/ionic-example$ ionic run android
Running command: /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/hooks/after_prepare/010_add_platform_class.js /home/jqrgen/stripe-checkout-nodejs-server/ionic-example
add to body class: platform-android
Running command: /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/cordova/run 
ANDROID_HOME=/home/jqrgen/android-sdk-linux
JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
WARNING : No target specified, deploying to device 'BH9013N81L'.
Running: /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/gradlew cdvBuildDebug -b /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build.gradle -PcdvBuildArch=arm -Dorg.gradle.daemon=true
:preBuild
:compileDebugNdk UP-TO-DATE
:preDebugBuild
:checkDebugManifest
:CordovaLib:compileLint
:CordovaLib:copyDebugLint UP-TO-DATE
:CordovaLib:mergeDebugProguardFiles UP-TO-DATE
:CordovaLib:preBuild
:CordovaLib:preDebugBuild
:CordovaLib:checkDebugManifest
:CordovaLib:prepareDebugDependencies
:CordovaLib:compileDebugAidl UP-TO-DATE
:CordovaLib:compileDebugRenderscript UP-TO-DATE
:CordovaLib:generateDebugBuildConfig UP-TO-DATE
:CordovaLib:generateDebugAssets UP-TO-DATE
:CordovaLib:mergeDebugAssets UP-TO-DATE
:CordovaLib:generateDebugResValues UP-TO-DATE
:CordovaLib:generateDebugResources UP-TO-DATE
:CordovaLib:packageDebugResources UP-TO-DATE
:CordovaLib:processDebugManifest UP-TO-DATE
:CordovaLib:processDebugResources UP-TO-DATE
:CordovaLib:generateDebugSources UP-TO-DATE
:CordovaLib:compileDebugJava UP-TO-DATE
:CordovaLib:processDebugJavaRes UP-TO-DATE
:CordovaLib:packageDebugJar UP-TO-DATE
:CordovaLib:compileDebugNdk UP-TO-DATE
:CordovaLib:packageDebugJniLibs UP-TO-DATE
:CordovaLib:packageDebugLocalJar UP-TO-DATE
:CordovaLib:packageDebugRenderscript UP-TO-DATE
:CordovaLib:bundleDebug UP-TO-DATE
:prepareAndroidCordovaLibUnspecifiedDebugLibrary UP-TO-DATE
:prepareDebugDependencies
:compileDebugAidl UP-TO-DATE
:compileDebugRenderscript UP-TO-DATE
:generateDebugBuildConfig UP-TO-DATE
:generateDebugAssets UP-TO-DATE
:mergeDebugAssets UP-TO-DATE
:generateDebugResValues UP-TO-DATE
:generateDebugResources UP-TO-DATE
:mergeDebugResources UP-TO-DATE
:processDebugManifest UP-TO-DATE
:processDebugResources      Unable to add '/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/assets/debug/www/bower_components/angular-stripe-checkout/angular-stripe-checkout.min.js.gz': file already in archive (try '-u'?)
ERROR: unable to process assets while packaging '/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/res/resources-debug.ap_'
ERROR: packaging of '/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/res/resources-debug.ap_' failed
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugResources'.
> com.android.ide.common.internal.LoggedErrorException: Failed to run command:
  	/home/jqrgen/android-sdk-linux/build-tools/22.0.1/aapt package -f --no-crunch -I /home/jqrgen/android-sdk-linux/platforms/android-22/android.jar -M /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/manifests/full/debug/AndroidManifest.xml -S /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/res/debug -A /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/assets/debug -m -J /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/generated/source/r/debug -F /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/res/resources-debug.ap_ --debug-mode --custom-package com.ionicframework.ionicexample803434 -0 apk --output-text-symbols /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/symbols/debug
  Error Code:
  	1
  Output:
  	      Unable to add '/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/assets/debug/www/bower_components/angular-stripe-checkout/angular-stripe-checkout.min.js.gz': file already in archive (try '-u'?)
  	ERROR: unable to process assets while packaging '/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/res/resources-debug.ap_'
  	ERROR: packaging of '/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build/intermediates/res/resources-debug.ap_' failed


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 3.376 secs

/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/cordova/node_modules/q/q.js:126
                    throw e;
                          ^
Error code 1 for command: /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/gradlew with args: cdvBuildDebug,-b,/home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/build.gradle,-PcdvBuildArch=arm,-Dorg.gradle.daemon=true
ERROR running one or more of the platforms: Error: /home/jqrgen/stripe-checkout-nodejs-server/ionic-example/platforms/android/cordova/run: Command failed with exit code 1
You may not have the required environment or OS to run this project

```

**Comment by jQrgen** [comment]
I managed to get ionic run android to work again when i deleted.

bower_components/angular-stripe-checkout/angular-stripe-checkout.min.js.gz

**Comment by tobyn** [comment]
Should be fixed in 4.0.1. Thanks.
